routemaster/jobs/fail.rb

```diff
@@ -0,0 +1,15 @@
+require 'routemaster/mixins/log'
+
+module Routemaster
+  module Jobs
+    class Fail
+      include Mixins::Log
+
+      def call(*args)
+        raise 'failing job'
+      end
+    end
+  end
+end
+
```

Comment:
Not easily. Job loading is conventional - it loads the job class using `require "routemaster/jobs/#{name}"`.
Any alternate suggestions?